include/zephyr/dt-bindings/adc/stm32_adc.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023 STMicrelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_STM32_ADC_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_STM32_ADC_H_
+
+#define STM32_ADC_REG_MASK		0xFFU
```

Comment:
You could use `BIT_MASK(8)` and so on.  This macro is already there for Zephyr ADC bindings in `include/zephyr/dt-bindings/adc/adc.h`.  But maybe this header is annoying to include here and we certainly all know what a bitmask in hex is.

Reply:
Done